pandas/tests/frame/test_query_eval.py

```diff
@@ -165,7 +164,9 @@ def test_eval_resolvers_as_list(self):
 
 class TestDataFrameQueryWithMultiIndex(tm.TestCase):
 
-    def check_query_with_named_multiindex(self, parser, engine):
+    @pytest.mark.parametrize('parser', ['pandas'])
```

Comment:
True.  I suspected that was the rationale.  I don't mind too much at all.  It helps make the test more flexible that way.  However, it may not be so for other developers, that's all.